openapi/java.xml

```diff
@@ -37,19 +38,17 @@
                 <artifactId>client-java</artifactId>
                 <artifactVersion>1.0-SNAPSHOT</artifactVersion>
                 <sourceFolder>src/main/java</sourceFolder>
-                <localVariablePrefix></localVariablePrefix>
                 <serializableModel>false</serializableModel>
                 <bigDecimalAsString>false</bigDecimalAsString>
                 <fullJavaUtil>false</fullJavaUtil>
                 <hideGenerationTimestamp>false</hideGenerationTimestamp>
                 <dateLibrary>joda</dateLibrary>
                 <useRxJava>false</useRxJava>
                 <library>okhttp-gson</library>
-                <type-mappings>intstr.IntOrString=IntOrString,resource.Quantity=Quantity</type-mappings>
-                <import-mappings>
-                  IntOrString=io.kubernetes.client.custom.IntOrString,Quantity=io.kubernetes.client.custom.Quantity,V1Patch=io.kubernetes.client.custom.V1Patch
-                </import-mappings>
+                <useReflectionEqualsHashCode>true</useReflectionEqualsHashCode>
               </configOptions>
+              <typeMappings>int-or-string=IntOrString,quantity=Quantity</typeMappings>
```

Comment:
@yue9944882 I noticed that you've added https://github.com/kubernetes-client/gen/pull/117/files#diff-4b08c5154861f4aefe8a9bc110c4a15bR155 to map quantity to Quantity so I think we can remove `,quantity=Quatity` here (but I've not tested it)

Reply:
this is still necessary. The code you linked is patching the swagger spec to add a format field to the `Quantity` definition, so we can install a custom mapping here.

Reply:
👌